packages/docs/src/components/app/Toc.vue

```diff
@@ -7,7 +7,6 @@
     location="right"
     width="256"
     floating
-    sticky
```

Comment:
should this only be removed for API pages? we don't want to remove it for the entire docs